src/main/java/engineering/swat/watch/Watch.java

```diff
@@ -192,16 +182,38 @@ public Watch onOverflow(Approximation whichFiles) {
         return this;
     }
 
+    private void validateOptions() throws IOException {
+        if (this.eventHandler == EMPTY_HANDLER) {
+            throw new IllegalStateException("There is no onEvent handler defined");
```

Comment:
```suggestion
            throw new IllegalStateException("There is no `on` handler defined");
```

(The method to set `this.eventHandler` is called `on` instead of `onEvent`)